debian/control

```diff
@@ -1,15 +1,15 @@
 Source: marker
-Section: unknown
+Section: gnome
 Priority: optional
 Maintainer: Olivier Bitsch <olivier.b@iabsis.com>
-Build-Depends: debhelper (>= 9), autotools-dev, libgtkspell3-3-dev, pandoc, libgtksourceview-3.0-dev, libgtk-3-dev (>= 3.20), libwebkit2gtk-4.0-dev, meson
+Build-Depends: debhelper (>= 9), autotools-dev, libgtkspell3-3-dev, libgtksourceview-3.0-dev, libgtk-3-dev (>= 3.20), libwebkit2gtk-4.0-dev, meson
 Standards-Version: 4.1.3
 Homepage: https://fabiocolacio.github.io/Marker/
 #Vcs-Browser: https://salsa.debian.org/debian/marker
 Vcs-Git: https://github.com/fabiocolacio/Marker
 
 Package: marker
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, libgtkspell3-3-dev, pandoc, libgtksourceview-3.0-dev, libgtk-3-dev (>= 3.20), libwebkit2gtk-4.0-dev
```

Comment:
I believe we should keep `pandoc` in the "Recommends:" section as it is not a hard dependency.